rest/kadai-rest-spring/src/test/java/io/kadai/task/rest/afterrequest/AfterRequestChangesIntTestProvider.java

```diff
@@ -0,0 +1,31 @@
+package io.kadai.task.rest.afterrequest;
+
+import io.kadai.common.api.KadaiEngine;
+import io.kadai.spi.task.api.AfterRequestChangesProvider;
+import io.kadai.task.api.models.Task;
+
+public class AfterRequestChangesIntTestProvider implements AfterRequestChangesProvider {
+  private KadaiEngine kadaiEngine;
+
+  @Override
+  public void initialize(KadaiEngine kadaiEngine) {
+    this.kadaiEngine = kadaiEngine;
+  }
+
+  @Override
+  public Task afterRequestChanges(Task task) throws Exception {
+    return task;
+  }
+
+  @Override
+  public Task afterRequestChanges(Task task, String workbasketId, String ownerId) throws Exception {
+    Task newTask = null;
+
+    try {
+      newTask = kadaiEngine.getTaskService().transferWithOwner(task.getId(), workbasketId, ownerId);
+    } catch (Exception e) {
+      System.out.println("Error getting task: " + e.getMessage());
```

Comment:
I know this is just Test Code, however I would really prefer a Logger. IMO System.out.println should never make it into final code